packages/wdio-config/src/utils.js

```diff
@@ -4,24 +4,27 @@ const DEFAULT_PROTOCOL = 'http'
 
 const REGION_MAPPING = {
     'us': '', // default endpoint
-    'eu': 'eu-central-1.'
+    'eu': 'eu-central-1.',
+    'eu-central-1': 'eu-central-1.',
+    'us-east-1': 'us-east1.'
 }
 
-export function getSauceEndpoint (region, isRDC) {
-    const dcRegion = REGION_MAPPING[region] ? region : 'us'
+export function getSauceEndpoint (region, isRDC, isHeadless) {
+    const shortRegion = REGION_MAPPING[region] ? region : 'us'
+    const product = isHeadless ? 'headless.' : ''
 
     if (isRDC){
-        return `${dcRegion}1.appium.testobject.com`
+        return `${shortRegion}1.appium.testobject.com`
```

Comment:
what does this `1` stands for?

Reply:
ah, got it. Is it always 1?

Reply:
for rdc our DC urls are
- eu1.appium.testobject.com
- us1.appium.testobject.com

This might change in the future but not sure in which direction yet.